tests/fixtures/module-unification-addon/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "foo",
+  "version": "0.0.0",
+  "description": "Generates an Ember addon with a module unification layout.",
+  "keywords": [
+    "ember-addon"
+  ],
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "repository": "",
+  "scripts": {
+    "build": "ember build",
+    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
+    "start": "ember serve",
+    "test": "ember test",
+    "test:all": "ember try:each"
+  },
+  "dependencies": {
+    "ember-cli-babel": "^6.6.0"
+  },
+  "devDependencies": {
+    "broccoli-asset-rev": "^2.4.5",
+    "ember-ajax": "^3.0.0",
+    "ember-cli": "~3.1.0-beta.1",
```

Comment:
it will fail each release

Reply:
The test is testing that the same files are being created, not that the contents of those files is the same. Afaik, all other tests are similar.

Reply:
oh, right. there is no verification of result files contents